pages/code-quality.md

```diff
@@ -21,6 +21,17 @@ If you use Maven, it has been automatically configured:
 
     ./mvnw -Pprod clean verify sonar:sonar -Dsonar.host.url=http://localhost:9001
 
+> <i class="fa fa-info-circle"></i> **Note for SonarQube 9.6.0 on Apple Silicon (M1)**
+>
+> SonarQube does not work with Apple Silicon using official images, a native image is not provided, and it fails in compatibility mode. You can build the Docker image locally to solve the problem: 
+>
+> ```
+> git clone git@github.com:SonarSource/docker-sonarqube.git
```

Comment:
the command fails with following error:
>➜  /tmp git clone git@github.com:SonarSource/docker-sonarqube.git
Cloning into 'docker-sonarqube'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

>Please make sure you have the correct access rights
and the repository exists.                                             

If I try using https, it works fine:
```git clone https://github.com/SonarSource/docker-sonarqube.git```

Reply:
Good catch @vishal423! PR to fix at https://github.com/jhipster/jhipster.github.io/pull/1230.